src/ds5/ds5-timestamp.cpp

```diff
@@ -256,6 +256,9 @@ namespace librealsense
             return 0;
         }
 
+        size_t pin_index = 0;
+        _has_metadata[pin_index] = true;
```

Comment:
What is the impact on IR stream?

Reply:
This change is done in the method: ds5_timestamp_reader_from_metadata_mipi_motion::get_frame_timestamp, so no impact on IR stream.